Scrapers/CompanyScrapers/NeuroMedScraper.py

```diff
@@ -0,0 +1,24 @@
+from bs4 import NavigableString
+
+from Scrapers.Scraper import Scraper
+
+
+class NeuroMedScraper(Scraper):
+    url = 'https://neuromedinc.com/careers/'
+    name = 'NeuroMed'
+
+    def scrape(self):
+        soup = self.scraping_unit(self.url)
+        for p_tag in soup.find_all('p'):
+            first_b = p_tag.find('b')
+            if first_b and "Job Openings" in first_b.get_text(strip=True):
+                siblings = list(first_b.next_siblings)
+                title = siblings[1][len("Position: "):]
```

Comment:
we can create `parsedSiblings`:
```suggestion
                parsedSiblings = list(map(lambda s: s.split(':')[-1], siblings))
                title = parsedSiblings[1]
                location = parsedSiblings[3]
                description = parsedSiblings[5]
```